internals/cli/cli.go

```diff
@@ -237,6 +238,11 @@ func Parser(cli *client.Client) *flags.Parser {
 	return parser
 }
 
+func applyPersonality(s string) string {
+	r := strings.NewReplacer("{{.ProgramName}}", cmd.ProgramName, "{{.DisplayName}}", cmd.DisplayName)
```

Comment:
Not that it matters much in this context, but the replacer would normally be created once, as it can do (relatively) expense setup once:

```go
var personalityReplacer  = strings.NewReplacer(...)

func applyPersonality(s string) string {
    return personalityReplacer.Replace(s)
}
```

Reply:
Wouldn't this cause the function to not change behavior when `cmd.ProgramName` is modified dynamically?

Reply:
Ooh, yes, you're right -- my mistake.